### ISSUE TITLE
SVG <stop> element requires offset attribute

### ISSUE THREAD

**Issue by 06tron** [validator/validator#2111]
### Summary

The checker says offset is required, but the definition of the offset attribute at https://svgwg.org/svg2-draft/pservers.html#GradientStopAttributes gives an initial value.

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
<!DOCTYPE html>
<html lang=en>
<meta charset=utf-8>
<title>gradient stop without explicit offset</title>
<svg viewBox=0,0,2,2>
	<linearGradient id=g>
		<stop/>
		<stop stop-opacity=.3 offset=1 />
	</linearGradient>
	<rect fill=url(#g) width=1 height=1 />
</svg>
```

### Checker error messages

```shell
Error: Element stop is missing required attribute offset.
```

### Spec links/citations

html.spec.whatwg.org/#refsSVG links to svgwg.org/svg2-draft/

Specifically: https://svgwg.org/svg2-draft/pservers.html#StopElementOffsetAttribute

### Steps to reproduce

n/a

### Expected behavior

n/a

### Actual behavior

n/a
